### DISCUSSION TITLE
simple-machine-learning-explanation/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#20] [Announcements]
# simple-machine-learning-explanation/

انقدر که میگن یادگیری ماشین منظورشون چیه؟ بالاخره تو هوش‌مصنوعی دارید چیکار می‌کنید؟ ربات درست می‌کنید؟ دوست دارم قبل از اون بگم که ادعایی در این زمینه ندارم. ادعا رو اون کسی می‌تونه داشته باشه که حداقل ۴ ۵ سال تو یک حوزه‌ای کار کرده باشه و سرد و گرم‌اش رو چشیده باشه. قبل از اون هر چی هست، «‌خاک‌بازی» هست.
به هر حال.
هوش مصنوعی چند بار اتفاق افتاده که بخوام راجع به هوش‌مصنوعی به کسانی که هیچی در این حوزه نمی‌دونند توضیح بدم.

https://aprd.ir/simple-machine-learning-explanation/

**Comment by amirghh** [comment]
سلام
خیلی خوب بود ممنون

**Comment by MohammadYeganeFar** [comment]
سلام ، امیدوارم حالت خوب باشه محمد هستم و چند سوال  داشتم:
۱- برای یادگیری ماشین از صفر تا استخدام در حوزه یادگیری ماشین چه منابعی رو پیشنهاد میکنی(که استفاده کردی)؟مثلا چه اشخاص یا چنل هایی؟
۲- من خرم آباد هستم .بنظرت نقل مکان برای کار به تهران  چقدر ایده خوب یا بدی  میتونه باشه؟
۳- الان مشغول کار توی شرکت خاصی هستی؟اگر آره چه سمتی و به طور کلی با چه اهدافی ؟
امیدوارم سوالام حوصله تو سر نبرده باشن.اگر تکرای هستن یا جایی بهشون پرداختی بهم بگو،ممنون.

**Reply by pourmand1376** [reply]
سلام محمد جان. 

ببخشید که با تأخیر بسیار زیاد جواب میدم. اتفاقاً سؤالات بسیار مهمی پرسیدی. چیزهایی که مدت‌ها ذهن من رو هم درگیر کرده بودند. 

من سعی کردم در پست جدیدم به سؤال دومت پاسخ بدم. 

https://aprd.ir/tehran-vs-yazd/

امیدوارم مفید باشه.
